### PR TITLE
We no longer use the acount_sessions table so we can drop it

### PR DESCRIPTION
**Pull request by elliott-davis**
tenor-22099232
Signed-off-by: Travis Elliott Davis <edavis@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/builder-sessionsrv/src/migrations/sessions.rs [hunk 1]
```diff
@@ -120,5 +123,47 @@ pub fn migrate(migrator: &mut Migrator) -> SrvResult<()> {
...
+
+    // Cleanup after table deprecation
+    migrator.migrate(
+        "accountsrv",
+        "DROP TABLE IF EXISTS account_sessions CASCADE",
+    )?;
+    migrator.migrate(
+        "accountsrv",
+        r#"DROP FUNCTION IF EXISTS insert_account_session_v2(
+                    a_account_id bigint,
```
Comment (raskchanky) [inline components/builder-sessionsrv/src/migrations/sessions.rs]: I dunno if you care, but technically you only need the type here, not the name.  That applies to all these DROP statements.

File: components/builder-sessionsrv/src/migrations/sessions.rs [hunk 2]
```diff
@@ -12,6 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
+// DO NOT USE THIS TABLE - IT HAS BEEN DEPRECATED AND DROPPED
```
Comment (chefsalim) [inline components/builder-sessionsrv/src/migrations/sessions.rs]: You probably don't need this comment at the top - either remove or bring it closer to the actual table it refers to..

Comment (elliott-davis) [reply components/builder-sessionsrv/src/migrations/sessions.rs]: I was looking for a way to signify that this file should no longer be used for migrations.

Comment (chefsalim) [reply components/builder-sessionsrv/src/migrations/sessions.rs]: Maybe a better note would be a TBD to delete this file entirely after migration cleanup is done..

Comment (raskchanky) [reply components/builder-sessionsrv/src/migrations/sessions.rs]: I like that idea.

**Comment by chefsalim** [comment]
tenor-217028866

**Comment by chefsalim** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/316489889).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/316489889).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by raskchanky** [comment]
tenor-152195471

**Pull request merged**